tests/unit/remote/test_s3.py

```diff
@@ -0,0 +1,89 @@
+import boto3
+import os
+import pytest
+from moto import mock_s3
+
+from dvc.remote.s3 import RemoteS3
+
+
+@pytest.fixture
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
```

Comment:
Are these needed? We don't have them in other moto tests.

Side note: you are modifying os.environ directly, which is dangerous and wrong. Should've used mocker or something.

Reply:
@efiop , it is not required but looks like it is a good practice to have if you want to make sure you are not using _real infrastructure_.

https://github.com/spulec/moto#how-do-i-avoid-tests-from-mutating-my-real-infrastructure

Agree with the side note, mocking the environment should work better :)